projects/services/consensus-service-typescript/src/consensus-server.ts

```diff
@@ -76,12 +81,14 @@ export default function (nodeTopology: any, env: any) {
   consensusConfig.nodeName = NODE_NAME;
   consensusConfig.clusterSize = Number(NUM_OF_NODES);
   consensusConfig.signBlocks = toLower(CONSENSUS_SIGN_BLOCKS) === "true";
-  consensusConfig.keyManager = consensusConfig.signBlocks ? new KeyManager({
+  consensusConfig.signBlocks ? new KeyManager({
     privateKeyPath: "/opt/orbs/private-keys/block/secret-key"
```

Comment:
Your changes to secret key management break the behaviour of benchmark consensus.